install-caddy.ps1

```diff
@@ -0,0 +1,112 @@
+﻿#!/usr/bin/env pwsh
```

Comment:
These two test files (`install-caddy.{sh,ps1}`) and the `install.bat` should be removed from the commit.

You can either make another commit either
- do a `git rebase -i 2` to squash them away
- or you can do a `git reset HEAD~1` and re-commit without them

(either way you'll need to `git push --force` afterwards to overwrite the commits that have them)